public/Lab-Files/Lesson02/Assignment/resume.html

```diff
@@ -0,0 +1,92 @@
+<!doctype html>
+ <html lang="en">
+ <head>
+     <meta charset="UTF-8">
+    
+       
+        <title>Priscilla's Resume</title>
+        
+        </head>
+        
+    <body
+   
+        
+        style="background-color: DarkSeaGreen">
+      
+        
+     
+        
+        
+        
+        <h4 style="color:HotPink"> Priscilla A. Avitua  <br/>
+       <img src="./resume/images/IMG02.JPG" alt="Headshot of Priscilla Avitua" style= "float:center;width:155px;"> <br/>
+<br/>
+<h4 style="font-family:Cursive">
+
+San Antonio TX, 78250<br/>
+Mobile Phone:512-501-0200<br/>
+E-mail:pavitua@gmail.com </h4> <br/> 
+
+<br/>
+  <hr>
+
+<div> <h3 style="font-family: Copperplate">Abjective: <br/>Seeking new and exciting company to expand my career path in Web Development.</h3></div>
+
+<img src="./resume/images/img03.jpg" alt="Computer with keyboard and mouse" >
+
+
+<p> <h4 style="color:CornflowerBlue;"><em>Aetna Healthcare Insurance<em> <br/> <a href="https://www.aetna.com/">Aenta</a> <br/>August 2014 - June 2015 <br/>
```

Comment:
Avoid using a heading inside a paragraph. Leave the heading outside. I also don't know where you closed the h4 tag.